src/credentials_rotators/npm-token-rotation/src/lambda/utils/npm-utils.ts

```diff
@@ -0,0 +1,66 @@
+import TOTP from "totp-generator";
+import axios from "axios";
+export const generateOtp = (seed: string) => {
+  return TOTP(seed, { digits: 6 });
+};
+
+export const createAccessToken = async (
+  username: string,
+  password: string,
+  otpSeed: string
+): Promise<string> => {
+  const otp = generateOtp(otpSeed);
+  const result = await axios.post(
```

Comment:
for all methods in this file: If the API calls fail, we might need try-catch block around this with appropriate failure log?